### PR TITLE
CLOUD-4080 - EAP XP 4.0 Images

### PR DESCRIPTION
**Pull request by luck3y**
https://issues.redhat.com/browse/CLOUD-4080
Signed-off-by: Ken Wills <kwills@redhat.com>

### CODE REVIEW

File: jboss/container/eap/galleon/module.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ description: Install Galleon descriptions and EAP s2i feature-pack. Configure ga
 
 envs:
 - name: S2I_FP_VERSION
-  value: "23.0.0.Final"
+  value: "26.0.0.Final"
```
Comment (jfdenise) [inline jboss/container/eap/galleon/module.yaml:8]: It should be 4.0.0.Final for Xp4, no?

File: jboss/container/eap/setup/module.yaml [hunk 2]
```diff
@@ -5,7 +5,3 @@ description:  "JBoss Enterprise Application Platform setup JBOSS_HOME"
 
 execute:
 - script: configure.sh
-
-envs:
```
Comment (jfdenise) [inline jboss/container/eap/setup/module.yaml:9]: Why are you removing this env variable?

**Comment by luck3y** [comment]
I think we thought that for XP3 as well, but ended up changing it back.
I'll check, but you probably know better than me :)

**Comment by luck3y** [comment]
It fails boot saying elytron is required, and this variable can not be set.
Should we not be using wilfly-cekit-modules 0.26.0?

File: jboss/container/eap/galleon/module.yaml (hunk 1 above)
Comment (luck3y) [reply jboss/container/eap/galleon/module.yaml]: +1 fixed, thanks -- not sure why I was confused there.

File: jboss/container/eap/setup/module.yaml (hunk 2 above)
Comment (luck3y) [reply jboss/container/eap/setup/module.yaml:9]: fixed

**Pull request merged**